### ISSUE TITLE
训练是 ctcloss爆炸

### ISSUE THREAD

**Issue by apple2333cream** [BADBADBADBOY/pgnet.pytorch#4]
ctc loss tensor(inf, device='cuda:0')
0/600/0/2000 | loss:inf | score_loss:0.82 | border_loss:0.56 | direction_loss:0.42 | ctc_loss:inf |  | Lr:0.00100000
/home/wzp/project/torch/lib/python3.6/site-packages/torch/nn/functional.py:1569: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
ctc loss tensor(nan, device='cuda:0')
ctc loss tensor(nan, device='cuda:0')
ctc loss tensor(nan, device='cuda:0')

**Comment by fxwfzsxyq** [comment]
> ctc loss tensor(inf, device='cuda:0') 0/600/0/2000 | loss:inf | score_loss:0.82 | border_loss:0.56 | direction_loss:0.42 | ctc_loss:inf | | Lr:0.00100000 /home/wzp/project/torch/lib/python3.6/site-packages/torch/nn/functional.py:1569: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead. warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.") ctc loss tensor(nan, device='cuda:0') ctc loss tensor(nan, device='cuda:0') ctc loss tensor(nan, device='cuda:0')

建议用paddleocr的英文数据集先跑跑看 可能是数据处理的问题
